doc/option_params/octal_integer.rb

```diff
@@ -0,0 +1,7 @@
+require 'optparse'
+include OptionParser::Acceptables
```

Comment:
Curious: Does this mixin load OctalInteger? Is it needed for the other examples, too?

Reply:
Yes.  I've include it for these three.

In optparse.rb:
```
  module Acceptables
    const_set(:DecimalInteger, OptionParser::DecimalInteger)
    const_set(:OctalInteger, OptionParser::OctalInteger)
    const_set(:DecimalNumeric, OptionParser::DecimalNumeric)
  end
```